packages/docusaurus/bin/docusaurus.js

```diff
@@ -254,6 +254,6 @@ async function run() {
 run();
 
 process.on('unhandledRejection', (err) => {
-  logger.error(err.stack);
```

Comment:
we have error.stack logged in multiple other places of the codebase, should we clean those up too?

Does it work correctly with toString/string interpolation logs?

Not sure why exactly, I think Node.js didn't log properly in older versions? 🤷‍♂️

Reply:
I don't know why either. Errors can both be stringified and logged very nicely. And yeah, I'll probably clean up the rest later